src/mineserver.cpp

```diff
@@ -207,8 +207,13 @@ Mineserver::Mineserver(int args, char **argarray)
       break;
     }
   }
-  
+
+// VS2012 debug wouldn't be able to find config file without this
```

Comment:
This sounds like a problem in the project file for Visual Studio as this will affect non-MSVC.

Do 'Release' builds have the same problem as 'Debug' builds?